VergeiOS/Wallet/WalletClient.swift

```diff
@@ -447,6 +449,12 @@ public class WalletClient {
         print("With signature: \(signature)")
         print("And Copayer id: \(copayerId)")
         
+        if referencedUrl.contains("/v1/balance/") {
+            self.cachedBalanceCredentials.balanceUrl = url.absoluteString;
+            self.cachedBalanceCredentials.copayerId = copayerId;
+            self.cachedBalanceCredentials.balanceSignature = signature;
+        }
+        
```

Comment:
Hmm I'm not a fan of this one. 🤔 
Maybe for now add a boolean to the function defaults to false.
But this might need a little refactor later.